packages/pagination/src/utils/shouldReturnObj.ts

```diff
@@ -0,0 +1,9 @@
+const shouldReturnObj = (condition: boolean, obj: any) => {
```

Comment:
As we've spoken, I've moved the function to the file where it's used, since its application is too tight with the component logic, this way hopefully makes more apparent what it does